command/login.go

```diff
@@ -240,6 +240,10 @@ func (c *LoginCommand) Run(args []string) int {
 		c.UI.Warn(wrapAtLength("A login request was issued that is subject to "+
 			"MFA validation. Please make sure to validate the login by sending another "+
 			"request to sys/mfa/validate endpoint.") + "\n")
+
+		// We return early to prevent success message from being printed
```

Comment:
which code path was being hit before this return?

Reply:
In particular, line 329 - if it continued then it assumes that it is successful.